src/screens/Assets/collectibles/CollectibleListItem.js

```diff
@@ -45,7 +45,7 @@ function CollectibleListItem({ title, iconUrl, onPress, width }: Props) {
       <IconContainer width={width}>
```

Comment:
Because RNRemoveSvg is using webview, this might be a potential bottleneck, since this component is a list item, which might cause perf issues for users with many NFTs. @vignesha22 could your try to check performance for user having say 30 NFTs, by using mock data or repeating existing items enough times?

Reply:
yeah if we have more than 5 itself the app crashes so yeah you are right. Do you have any other solution rather than using webview?

Reply:
That's what I was afraid of. Seems like we do not have perfect solution here. Maybe we could use web-view high res for viewing single collectible in full screen view and use the RNSvg for list view.

@IAmKio, @poocart, @vignesha22  wdyt?

Reply:
That's the thing, it does not render as it supposed to using RNSvg. Agree with 2 libraries serving list and preview screens, but the main idea is to fix the broken ones and showing on the list broken is not a solution, maybe we can show webview svg only for broken ones identifying them by contract? That would narrow the list of using webview SVGs only.

Reply:
Also maybe is there any way to lower the perfomance of that webview svg lib?

Reply:
From my perspecting app crashing from memory exhaustion is worse then broken SVG. Webview path is unlikely to succeed in any list environment, because of webview memory footprint. I think we need to research some more solutions. Couple of ideas:
- some other RN SVG library
- caching low-res PNG renders of SVG for list usage (front end side or back end side)

Reply:
Was thinking about the latter too, but that's a significant amount of work. I see it residing in back-end, but that would require effort that is not cost-effective for these few NFTs to support. Also these Uniswap V3 SVGs are moving so still images is OK, but wouldn't completely deliver what expected and transforming those into GIFs is too much fuss for sure.

Reply:
If webview renders moving (animated?) SVG for whole screen viewing then we would not need to use backend rendering for that, just pre-render them for non-animated, lower-res list items.

Reply:
@vignesha22 thoughts?